activestorage/storage.py

```diff
@@ -1,55 +1,87 @@
-from ast import Not
+from numpy.ma import masked_equal, masked_greater, masked_less, is_masked
```

Comment:
code reads better if you just use the vanilla `import numpy as np` and then call `np.ma` modules down the code - don't worry about the import burden, as long as you don't do `from numpy import *` it's all good

Reply:
agreed, please change

Reply:
done